spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/ConfigMapConfigProperties.java

```diff
@@ -36,9 +35,9 @@
 
 	private boolean enableApi = true;
 
-	private List<String> paths = new LinkedList<>();
+	private List<String> paths = Collections.emptyList();
```

Comment:
it does not look like the code cares what type of `List` this holds, so why have it a `LinkedList` when it could be simply an empty one? I guess the point of initializing this with _something_ is so that the callers will never get a `null`, but an _empty_ List instead; in such a case let's make this explicit.